api/py/test/test_run.py

```diff
@@ -16,22 +16,33 @@
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
 
-from ai.chronon.repo import run
```

Comment:
Is this file also mostly linter changes?

Reply:
We added a new parameter for `download_jar()` and updated the unit tests accordingly in this file. For example: `def mock_cmd(url, path, skip_download):`